src/api/search-fulltext-api/suggestions.ts

```diff
@@ -0,0 +1,9 @@
+import { searchFullTextApiGet } from './host';
+
+export const getSearchSuggestions = (q: string, transport?: boolean) =>
+  searchFullTextApiGet(
+    `/suggestion?q=${encodeURI(q)}${transport ? '&transport=true' : ''}`
```

Comment:
I stedet for å legge parametre til i 'path' kan du utvide 'searchFullTextApiGet' til å ta params, og videre 'searchFullTextApi' til å kunne akseptere params. Da man gi med parametre fra der man bruker 'getSearchSuggestions'